com.github.hluk.copyq.yaml

```
@@ -23,9 +23,11 @@ modules:
       - -DCOPYQ_AUTOSTART=OFF
       - -DCOPYQ_AUTOSTART_COMMAND=flatpak run com.github.hluk.copyq
       - -DWITH_QT6=1
-    name: copyq
     sources:
-      - commit: 4aab252568347dbea3e8d7929d92ae2b7308aef2
-        tag: v9.1.0
-        type: git
+      - type: git
         url: https://github.com/hluk/CopyQ.git
+        tag: v9.1.0
+        commit: 4aab252568347dbea3e8d7929d92ae2b7308aef2
+        x-checker-data:
+          type: git
+          tag-pattern: ^v(\d\.\d\.\d)$'
```

It's missing a single quote:

```suggestion
          tag-pattern: '^v(\d\.\d\.\d)$'
```